scripts/dts/gen_defines.py

```diff
@@ -577,20 +577,20 @@ def out_dev(dev, ident, val, name_alias=None):
         aliases.append(dev_prefix + "_" + name_alias)
         aliases += [alias + "_" + name_alias for alias in dev_aliases(dev)]
 
-    out(dev_prefix + "_" + ident, val, aliases)
+    return out(dev_prefix + "_" + ident, val, aliases)
```

Comment:
Could mention this in the other doc comments too, and not just in `out()`.

Reply:
I thought that was covered by "Like out(), but..."  I can repeat it if you prefer.

Reply:
Didn't make the connection (though maybe I should've ;), so might be nice to explicitly mention the return value at the end for the likes of me.

Reply:
Hmm... where's that "Like out(), but..."? I don't see it.

Reply:
`out_dev_s` and `out_s` are `Like out(), but...`  I'll just add it.